corehq/apps/locations/bulk_management.py

```diff
@@ -817,7 +835,7 @@ def save_types(type_stubs, excel_importer=None):
     return all_objs_by_code
 
 
-def save_locations(location_stubs, types_by_code, excel_importer=None):
+def save_locations(location_stubs, types_by_code, domain, excel_importer=None):
     """
     :param location_stubs: (list) List of LocationStub objects with
         attributes like 'db_object', 'needs_save', 'do_delete' set
```

Comment:
nit: you are missing the `:param:` docstring for `domain` now (pretty obvious what it should be, but everything else is documented)